frontend/recipe/scrollgridssidebyside/grid-sync.js

```diff
@@ -0,0 +1,15 @@
+const grids = [];
+
+function addGrid(grid) {
+    grids.push(grid);
+    grid.$.outerscroller.addEventListener("scroll", _ => syncGrids(grid));
+}
+
+function syncGrids(source) {
+    grids.filter(grid => grid !== source)
+        .forEach(grid => requestAnimationFrame(()=>
+            grid.$.outerscroller.scrollTop = source.$.outerscroller.scrollTop));
```

Comment:
ping @marcushellberg scrolling the #outerscroller element did fix the flickering of the header for me